src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+       href="#logo"
+       class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo_image"
+          alt="Moyo Company Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_link">
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Here, `nav_link` seems to be more suitable for a link, not for a list. Consider changing it to `nav__list` or similar.